com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Common/AtmosphericScatteringRayTracing.hlsl

```diff
@@ -8,7 +8,11 @@ void ApplyFogAttenuation(float3 origin, float3 direction, float t, inout float3
     if (_FogEnabled)
     {
         float dist = min(t, _MaxFogDistance);
+#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING == 0)
         float absFogBaseHeight = GetAbsolutePositionWS(float3(0.0, _HeightFogBaseHeight, 0.0)).y;
```

Comment:
_HeightFogBaseHeight should also be handle on C# side like we do for rasterizatin:

            float crBaseHeight = baseHeight.value;

            if (ShaderConfig.s_CameraRelativeRendering != 0)
            {
                crBaseHeight -= hdCamera.camera.transform.position.y;
            }

            float layerDepth = Mathf.Max(0.01f, maximumHeight.value - baseHeight.value);
            float H = ScaleHeightFromLayerDepth(layerDepth);
            cb._HeightFogExponents = new Vector2(1.0f / H, H);
            cb._HeightFogBaseHeight = crBaseHeight;

Reply:
currently it is already handle, this _HeightFogBaseHeight  should be share with raytrace part? Or maybe we wait for the PR of Julien?

Reply:
corrected